doc/pull-request-checklist.md

```diff
@@ -0,0 +1,28 @@
+# Pull request (PR) checklist
+
+## For PR submitters
+
+The PR submitter should:
+
+-   [ ] Write or supplement test(s) for the file(s) you touch in your PR.
+-   [ ] Make sure Numberscope runs properly, including former supposedly
+        unmodified behaviors and newly implemented ones. In particular, run it
+        via 'npm run dev' and with the browser console open and make sure
+        there are no log messages from Numberscope code, warnings, or errors.
+
+## For PR reviewers
+
+-   [ ] All new or changed features are appropriately documented.
+-   [ ] Tests are appropriately modified for all new or changed features.
+-   [ ] The PR is passing lint by running npm run lint. There should be no
+        changed files and no warnings/errors.
+-   [ ] The PR builds by running npm run build. (This also checks type
+        correctness.) There should be no errors, and for now the only allowed
+        warning is the one about some assets being too big.
+-   [ ] The PR passes all tests. (Right now, just by running npm run
+        test:unit.)
```

Comment:
Ditto, what's the mechanism for updating this when there are more tests?  Maybe we should put a timestamp on these "right now" and "for now" comments?  At least that way, someone who is looking at the file can tell how stale they are.

Reply:
Added timestamp and created https://github.com/numberscope/frontscope/issues/174.